### PR TITLE
Day04 Changes

### PR DESCRIPTION
**Pull request by ArthurSoftvision1**

### CODE REVIEW

File: FirstProject/css/feedback.css [hunk 1]
```diff
@@ -89,15 +78,16 @@ h1.interview_text_area {
...
     }
 
     header form {
         margin-left: 0!important;
     }
 
     .header_form_section {
-        width: 94%!important;
+        width: 87%;
+        margin: 0 40px auto auto;
```
Comment (softvision-alexcarcea) [inline FirstProject/css/feedback.css:90]: Just making sure that this is what you want it to be: can you describe what this expands to?

**Review by softvision-alexcarcea — APPROVED** [review]

File: FirstProject/css/feedback.css (hunk 1 above)
Comment (ArthurSoftvision1) [reply FirstProject/css/feedback.css:90]: I set the width to 87% and I set the margin-top: 0;
margin-right: 40px; and margin-bottom && left: auto.

Comment (softvision-alexcarcea) [reply FirstProject/css/feedback.css:90]: I'm convinced! :smile:

Comment (ArthurSoftvision1) [reply FirstProject/css/feedback.css:90]: I typed the code like that and it worked for me. The code makes what I want :))

So.. :)) 
Is this technique wrong?

Comment (softvision-alexcarcea) [reply FirstProject/css/feedback.css:90]: Not at all. I just found it a bit odd that you had a `margin-bottom` of `auto` (top and bottom `auto` margins are [treated as 0](https://stackoverflow.com/questions/12415661/using-marginauto-to-vertically-align-a-div), by default), so I wanted to make sure that you had the right order. That seems to be the case, so I'm satisfied with the answer, irrespective of what the resulting layout would look like (if it looks OK to you, it's enough for me :wink:).

**Pull request merged**